### PR TITLE
add "(indiv)" children to nodes with non-negligible indivual time/alloc

### PR DESCRIPTION
**Pull request by nfrisby**
Fantastic tool. Thanks very much!

One behavior that I find burdensome when using the tool is that a node's individual time and alloc vanish when I expand it. This causes me some confusion.

This commit "fixes" that behavior (IMO) by adding a dummy child to each proper node. The new dummy node has the same individual time and alloc as its parent, so that that information persists in the visualization.

In other words, the size of the rectangles in the treemap now all use the same global scale no matter how many nodes have been expanded.

I invite others to clean-up this code: add comments, guard this behavior with a command-line switch, do something more reasonable than the "1000000" literal, etc.

Thanks again for a great tool!

### CODE REVIEW

**Comment by jaspervdj** [comment]
Fixed this in the latest release!
